pkg/ebpf/events_processor.go

```diff
@@ -343,6 +349,35 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 	case InitModuleEventID:
 		t.invokeIoctlTriggeredEvents()
 		t.updateKallsyms()
+	case HookedProcFopsEventID:
+		fopsAddresses, err := getEventArgUlongArrVal(event, "hooked_fops_pointers")
+		if err != nil || fopsAddresses == nil {
+			return fmt.Errorf("error parsing hooked_proc_fops args: %w", err)
+		}
+		hookedFops := make([]trace.HookedSymbolData, 0)
+		for idx, addr := range fopsAddresses {
+			inTextSeg, err := t.kernelSymbols.TextSegmentContains(addr)
+			if err != nil {
+				return fmt.Errorf("error checking kernel address: %v", err)
+			}
+			if !inTextSeg {
+				hookingFunction := parseSymbol(addr, t.kernelSymbols)
+				if hookingFunction.Owner == "system" {
```

Comment:
If we're here then we already know the symbol is not in the text section, isn't it enough?